python/ray/experimental/state.py

```diff
@@ -1364,6 +1374,7 @@ def available_resources(self):
                     if local_scheduler_id not in local_scheduler_ids:
                         del available_resources_by_id[local_scheduler_id]
         else:
+            # TODO(rliaw): Is this a fair assumption?
```

Comment:
Yes, this is a safe assumption. `self.redis_clients` has one client per shard, and the number of shards doesn't change.

Reply:
I'd remove this comment.